src/MoBi.Core/Services/ModuleConfigurationBuilder.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Linq;
+using MoBi.Core.Domain.Model;
+using OSPSuite.Core.Domain;
+using OSPSuite.Utility.Extensions;
+
+namespace MoBi.Core.Services
+{
+   public interface IModuleConfigurationBuilder
```

Comment:
Shouldn't we move this into some kind of converter namespace ?This is not something that should be used outside of conversion right?

Reply:
I'll move the existing one too ```ProjectClassificationConverter```